File: Selenium/Activities/Activity7.py
from selenium import webdriver
from selenium.webdriver.common.by import By

driver = webdriver.Chrome()
actions = webdriver.ActionChains(driver)

driver.get("https://v1.training-support.net/selenium/drag-drop")
print(driver.title)

element = driver.find_element(By.ID, "draggable")
element1 = driver.find_element(By.ID, "droppable")
element2 = driver.find_element(By.ID, "dropzone2")

actions.click(element).drag_and_drop(element, element1).perform()
print(element1.text)
if element1.text.find("Dropped"):
    print("ball dropped in dropzone1")

    actions.click(element).drag_and_drop(element1, element2).perform()
    print(element2.text)
    if element2.text.find("Dropped"):
        print("ball dropped in dropzone2")
else:
    print("check why ball was not dropped in dropzone1")

driver.quit() 

File: Selenium/Activities/Activity7.java
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.interactions.Actions;

public class Activity7 {
    public static void main(String[] args) {
        WebDriver driver = new FirefoxDriver();
        Actions builder = new Actions(driver);

        driver.get("https://v1.training-support.net/selenium/drag-drop");
        System.out.println(driver.getTitle());

        WebElement element = driver.findElement(By.id("draggable") );
        WebElement element1 = driver.findElement(By.id("droppable") );
        WebElement element2 = driver.findElement(By.id("dropzone2") );

        builder.click(element).dragAndDrop(element, element1).pause(2000).release().build().perform();
        if (element1.findElement(By.tagName("p")).getText().equals("Dropped!")) {
            System.out.println("ball dropped in dropzone1");
            builder.click(element).dragAndDrop(element1, element2).pause(2000).release().build().perform();
            if (element2.findElement(By.tagName("p")).getText().equals("Dropped!")){
                System.out.println("ball dropped in dropzone2");
            }
        else {
                System.out.println("check why ball was not dropped in dropzone1");
            }
        }

        driver.quit();


    }
}
